test/integration/script-loader/base/pages/page4.js

```diff
@@ -1,33 +1,45 @@
+/* global _ */
```

Comment:
In the production mode test, Next.js will perform eslint. Add a `global` for lodash.